src/main/java/org/jetbrains/bsp/bazel/server/bazel/BazelDataResolver.java

```diff
@@ -1,30 +1,32 @@
-package org.jetbrains.bsp.bazel.server.resolvers;
+package org.jetbrains.bsp.bazel.server.bazel;
 
 import com.google.common.collect.Iterables;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.List;
-import org.jetbrains.bsp.bazel.server.data.BazelData;
-import org.jetbrains.bsp.bazel.server.data.ProcessResults;
+import org.jetbrains.bsp.bazel.server.bazel.data.BazelData;
+import org.jetbrains.bsp.bazel.server.bazel.data.ProcessResults;
 
 public class BazelDataResolver {
+
   private final BazelRunner bazelRunner;
 
   public BazelDataResolver(BazelRunner bazelRunner) {
     this.bazelRunner = bazelRunner;
   }
 
   public BazelData resolveBazelData() {
-    String execRoot = readOnlyBazelLine("info", "execution_root");
-    String workspaceRoot = readOnlyBazelLine("info", "workspace");
-    String binRoot = readOnlyBazelLine("info", "bazel-bin");
+    String execRoot = readOnlyBazelLine("execution_root");
+    String workspaceRoot = readOnlyBazelLine("workspace");
+    String binRoot = readOnlyBazelLine("bazel-bin");
     Path workspacePath = Paths.get(execRoot);
     String workspaceLabel = workspacePath.toFile().getName();
     return new BazelData(execRoot, workspaceRoot, binRoot, workspaceLabel);
   }
 
-  private String readOnlyBazelLine(String... args) {
-    ProcessResults processResults = bazelRunner.runBazelCommand(args);
+  private String readOnlyBazelLine(String argument) {
+    ProcessResults processResults =
+        bazelRunner.commandBuilder().info().withArgument(argument).runBazel();
```

Comment:
The naming of this function does not seem intuitive. Perhaps `readBazelInfo` is better, wdyt?

Reply:
In general I had a problem with names there - I wanted to to do a mapping 1-1 from `bazel <command>` to `bazelRunner.commandBuilder().<command>`. Nevertheless I fully agree that it might be not enough intuitive. `readBazelInfo` is difficult to extend into the pattern (for `run` or `build`), maybe `infoCommand`? but then `runCommand` is a bit ambiguous? Any other ideas?

Reply:
Maybe `readBazelInfoLine`? Not sure if it doesn't suffer from the same problem

Reply:
I would like to use the same schema to all bazel commands - `run`, `build`, `info` ...

Reply:
maybe something like `infoFetch` then? sounds odd but along these lines maybe? definitely a more troublesome name.

Reply:
Maybe I wasn't clear - I would like to find such schema: `<schema><bazel command><schema>` for example `callBuild` / `callRun`, I think that `runFetch` doesn't sound good

Reply:
Ah, I got it now, sorry. Not sure I can think of a name for this one :thinking: ...